src/wallet/feebumper.cpp

```diff
@@ -234,14 +246,16 @@ Result CreateRateBumpTransaction(CWallet& wallet, const uint256& txid, const CCo
     // is one). If outputs vector is non-empty, replace original
     // outputs with its contents, otherwise use original outputs.
     std::vector<CRecipient> recipients;
-    for (const auto& output : outputs.empty() ? wtx.tx->vout : outputs) {
-        if (!OutputIsChange(wallet, output)) {
-            CRecipient recipient = {output.scriptPubKey, output.nValue, false};
-            recipients.push_back(recipient);
-        } else {
+    const auto& tx_outs = outputs.empty() ? wtx.tx->vout : outputs;
+    for (size_t i = 0; i < tx_outs.size(); ++i) {
+        const CTxOut& output = tx_outs.at(i);
+        if ((reduce_output.has_value() && reduce_output.value() == i) || (!reduce_output.has_value() && OutputIsChange(wallet, output))) {
```

Comment:
In https://github.com/bitcoin/bitcoin/commit/a902eaf5be4a83dff37f28a988e5f3b2425551c0:

Could write it prettier:
```suggestion
        if (reduce_output.has_value() ? reduce_output.value() == i : OutputIsChange(wallet, output)) {
```

Reply:
Done